xrpl/transaction/payment_test.go

```diff
@@ -474,54 +474,54 @@ func TestPayment_Flatten(t *testing.T) {
 					},
 				},
 				SendMax: types.IssuedCurrencyAmount{
-					Issuer:   "r3dFAtNXwRFCyBGz5BcWhMj9a4cm7qkzzn",
 					Currency: "USD",
+					Issuer:   "r3dFAtNXwRFCyBGz5BcWhMj9a4cm7qkzzn",
 					Value:    "3",
 				},
 			},
-			expected: `{
-				"Account": "rJwjoukM94WwKwxM428V7b9npHjpkSvif",
+			expected: FlatTransaction{
```

Comment:
What's the reason of changing the json format to `FlatTransaction`?

Reply:
I thought it's a nicer/simpler way to write test and in the end to get rid of this `CompareFlattenAndExpected` method. What do you think ?